u2f-tests/HID/u2f_util.h

```diff
@@ -112,6 +112,14 @@ int U2Fob_recv(struct U2Fob* device, uint8_t* cmd,
 // returns
 //   negative error
 //   positive sw12, e.g. 0x9000, 0x6985 etc.
+int U2Fob_exchange(struct U2Fob* device,
```

Comment:
Slightly confused by choice of function name and comment.  Would be nice if something about the comment and method name made it clear how this is different from ```U2Fob_apdu()```.